app/src/main/res/layout/feedback.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <Button
+        android:id="@+id/send_email_btn"
+        android:text="SEND"
+        android:layout_alignParentRight="true"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:onClick="send_email" />
+
+    <TextView
+        android:id="@+id/enter_textview"
+        android:textSize="40dp"
+        android:text="@string/enter_text_here"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content" />
+
+    <EditText
```

Comment:
There's no positioning on the edittext, and messages with many lines of text will overlap the TextView/Button